src/components/generic/Button.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Button as AntdButton } from 'antd';
+import theme from '@/theme/theme';
+
+interface ButtonProps {
+  icon?: React.ReactNode;
+  onClick: () => void;
+  children?: React.ReactNode;
+}
+
+const Button: React.FC<ButtonProps> = ({ icon, onClick, children }) => {
+  const buttonStyle: React.CSSProperties = {
+    borderRadius: theme.borderRadius.m,
```

Comment:
We should define all of our theme properties under Antd and use their `useToken()`  hook to retrieve the theme properties. This will allow us to avoid having duplicate theme entries like colors, spacing, border radius, which are all defined in our Antd theme already.

```suggestion
import { theme } from 'antd';

const { useToken } = theme;

const Button: React.FC<ButtonProps> = ({ icon, onClick, children }) => {
  const { token } = useToken();
  
  const buttonStyle: React.CSSProperties = {
    borderRadius: token.borderRadius,
```

Reply:
Keeping it like this as discussed in the team meeting